src/GitHub.Exports/Models/IPullRequestReviewModel.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace GitHub.Models
+{
+    /// <summary>
+    /// The possible states of a pull request review.
+    /// </summary>
+    public enum PullRequestReviewState
+    {
+        /// <summary>
+        /// A review that has not yet been submitted.
+        /// </summary>
+        Pending,
+
+        /// <summary>
+        /// An informational review.
+        /// </summary>
+        Commented,
+
+        /// <summary>
+        /// A review allowing the pull request to merge.
+        /// </summary>
+        Approved,
+
+        /// <summary>
+        /// A review blocking the pull request from merging.
+        /// </summary>
+        ChangesRequested,
+
+        /// <summary>
+        /// A review that has been dismissed.
+        /// </summary>
+        Dismissed,
+    }
+
+    /// <summary>
+    /// Represents a review of a pull request.
+    /// </summary>
+    public interface IPullRequestReviewModel
+    {
+        /// <summary>
+        /// Gets the ID of the review.
+        /// </summary>
+        long Id { get; }
+
+        /// <summary>
+        /// Gets the GraphQL ID for the review.
+        /// </summary>
+        string NodeId { get; set; }
```

Comment:
Aha! So Node Id is for GraphQL. Why is that different from the Id of the review though still?

Reply:
Oh got it - that makes sense for sure :) Thanks!